tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -287,7 +251,7 @@
                 <ComboBox x:Uid="ProviderSelectionComboBox"
                           x:Name="ProviderSelectionComboBox"
                           Margin="0 3 0 0"
-                          MinWidth="180"

```

Comment:
Is this change for resolving a different issue? If yes, consider updating the PR description.

Reply:
I was not aware I made this change.  IT will be reverted.